modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -223,6 +224,12 @@ public ObjectDefinition postObjectDefinition(
 			ObjectDefinition objectDefinition)
 		throws Exception {
 
+		if (Validator.isNull(objectDefinition.getModifiable()) &&
```

Comment:
I think this condition is wrong, did you have a chance to test it through REST API?